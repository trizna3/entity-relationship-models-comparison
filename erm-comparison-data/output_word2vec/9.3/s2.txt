contracts-contracts
payments-payments
students-students
buildings-addresses;buildings;floors
rooms-rooms
account-accounts
CONTRACT_11_ASSOCIATION:S-addresses-buildings
CHANGE_CARDINALITY:S-floors-buildings
CONTRACT_11_ASSOCIATION:S-buildings-floors
