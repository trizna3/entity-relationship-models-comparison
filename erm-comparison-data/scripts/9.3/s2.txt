entity set addresses {
	"town"
	"street"
	"number"
};

entity set buildings {
	"name"
};

entity set floors {
	"number"
};

entity set rooms {
	"capacity"
	"fee"
};

entity set students {
	"name"
	"age"
};

entity set contracts {
	"start"
	"end"
};

entity set payments {
	"date"
	"time"
	"sum"
};

entity set accounts {
	"credit"
};


association between 
{
	addresses ONE 
	buildings ONE
}{};

association between 
{
	buildings ONE
	floors MANY
}{};

association between 
{
	floors ONE 
	rooms MANY
}{};

association between 
{
	rooms ONE 
	contracts MANY
}{};

association between 
{
	contracts ONE 
	payments MANY
}{};

association between 
{
	contracts MANY
	students ONE
}{};

association between 
{
	students ONE 
	accounts ONE
}{};

association between 
{
	payments MANY
	accounts ONE
}{};