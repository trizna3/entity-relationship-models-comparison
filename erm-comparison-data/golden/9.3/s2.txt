buildings-buildings
rooms-rooms
contracts-contracts
payments-payments
account-accounts
students-students